app/models/event_user.rb

```diff
@@ -1,6 +1,6 @@
 class EventUser < ActiveRecord::Base
   # TODO Do we need these roles?
-  ROLES = [['Speaker', 'speaker'], ['Submitter', 'submitter'], ['Moderator', 'moderator']]
+  ROLES = [%w[Speaker speaker], %w[Submitter submitter], %w[Moderator moderator]]
```

Comment:
@differentreality why do you want to drop it?

What I meant with my commit is that is `%w()` and not `%w[]`

But even if you don't like the `%w()` syntax we can also enable the cop to enforce using an array with the strings. I would prefer to use `%w()`, because it is the default, but the other options is also fine. Enabling it we will ensure that we are consistent and we always use the same syntax.

https://rubocop.readthedocs.io/en/latest/cops_style/#stylewordarray